src/google_apps_clj/google_sheets_v4.clj

```diff
@@ -240,8 +241,9 @@
                                     (.setGridProperties
                                      (-> (GridProperties.)
                                          (.setRowCount (int (count rows)))
-                                         (.setColumnCount (int num-cols)))))))))
-                      (-> (Request.)
+                                          (.setColumnCount (int num-cols)))))))))]
+                      (when (< 0 (count rows))
```

Comment:
y not `empty?`?  is that function unpredictable / worse than `seq` for some reason?